ui/src/routes/devices.$id.settings.tsx

```diff
@@ -19,7 +19,7 @@ import { LinkButton } from "../components/Button";
 import { cx } from "../cva.config";
 import { useUiStore } from "../hooks/stores";
 import useKeyboard from "../hooks/useKeyboard";
-import { useResizeObserver } from "../hooks/useResizeObserver";
+import { useResizeObserver } from "usehooks-ts";
```

Comment:
use the **already included** `usehook-ts` implementations and kill off our versions